tasks/setup.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import io
+import os
+
+import setuptools
+
+name = 'google-cloud-cloudtasks'
+description = 'Cloud Tasks API API client library'
+version = '0.1.0'
+release_status = 'Development Status :: 3 - Alpha'
+dependencies = [
+    'google-api-core[grpc] >= 1.1.0, < 2.0.0dev',
+    'enum34; python_version < "3.4"',
+    'grpc-google-iam-v1<0.12dev,>=0.11.4',
+]
+
+package_root = os.path.abspath(os.path.dirname(__file__))
+
+readme_filename = os.path.join(package_root, 'README.rst')
+with io.open(readme_filename, encoding='utf-8') as readme_file:
+    readme = readme_file.read()
+
+packages = [
+    package for package in setuptools.find_packages()
+    if package.startswith('google')
+]
+
+namespaces = ['google']
```

Comment:
Uh-oh, we're missing the `google.cloud` namespace here. Can you make sure we didn't forget this on any other packages?

Reply:
locally they all seem to use these lines. I searched for `namespaces =` and 30 results came back. All using the following:

```
# Determine which namespaces are needed.
namespaces = ['google']
if 'google.cloud' in packages:
    namespaces.append('google.cloud')
```